docs/operator/configuration_parameters.md

```diff
@@ -532,36 +532,48 @@ and only contains information about the specific RSE's for the Rucio instance.
 Read more about RSEs [here](../started/concepts/rucio_storage_element.md)
 and how to set them up [here](./configuration.md#creating-new-rses).
 
-- **associated_sites**: <!--??--> Separated by commas. Default: `None`.
-- **auto_approve_bytes**: Auto Approve Limit. Example: `500GB`. No default.
+- **associated_sites**: <!--??--> String. Separated by commas. Used for chaining of subscriptions so that transfers to one RSE will also be mirrored to associated_sites. No default.
```

Comment:
The `<!--??-->` doesn't seem to do anything, so now may be the best time to just take them out to avoid confusion.